### PR TITLE
Create xAB Functional Content.py

### PR DESCRIPTION
**Pull request by Sophie-Pl**

### CODE REVIEW

File: plugins/Python4Capella/sample_scripts/xAB Functional Content.py [hunk 1]
```diff
@@ -0,0 +1,214 @@
...
+# script-type          : Python
+# description          : Extract content of xAB diagram
+# popup                : enableFor(org.eclipse.sirius.viewpoint.DRepresentationDescriptor)
+
+'''
+This script allows to exxtract content of xAB diagram
+It will create a folder result in the selected Capella project with the resulting xlsx file.
+'''
+
+# To run it:
```
Comment (ylussaud) [inline plugins/Python4Capella/sample_scripts/xAB Functional Content.py:11]: To run it you need to right click on a DRepresentationDescriptor (diagram in the project explorer ?) and select the "Extract content of xAB diagram" menu.

File: plugins/Python4Capella/sample_scripts/xAB Functional Content.py [hunk 2]
```diff
@@ -0,0 +1,214 @@
+# name                 : Extract content of xAB diagram
+# script-type          : Python
+# description          : Extract content of xAB diagram
+# popup                : enableFor(org.eclipse.sirius.viewpoint.DRepresentationDescriptor)
+
+'''
+This script allows to exxtract content of xAB diagram
```
Comment (ylussaud) [inline plugins/Python4Capella/sample_scripts/xAB Functional Content.py:7]: exxtract -> extract

**Review by ylussaud — CHANGES_REQUESTED** [review]
Small changes needed in the comments.

**Review by Sophie-Pl — COMMENTED** [review]
Ok for the modifications to add in comments

**Comment by Sophie-Pl** [comment]
I reopen the pull request in order to merge the script xAB functional content (update after review)

**Pull request merged**